### ISSUE TITLE
macOS 14.4 kickstart usage

### ISSUE THREAD

**Issue by amyers22** [kcrawford/dockutil#162]
As of macOS 14.4, the usage of the `launchctl kickstart` command is limited. When using dockutil on a Mac with macOS 14.4 and not using the `--no-restart` command will cause an error: `Could not kickstart service "com.apple.Dock.agent": 1: Operation not permitted`. 

Is it possible to move away from the usage of `launchctl kickstart` and instead use `kill`. Other notes of this issue are documented below, but more targeted toward the `softwareupdated` command.

https://www.kevinmcox.com/2024/03/changes-to-launchctl-kickstart-in-macos-14-4/

**Comment by kcrawford** [comment]
I think this was already fixed in latest release. Let me know if that resolves it.

**Comment by ivankovnatsky** [comment]
Yes, it does work with the latest version, I think the issue could be closed.
